scripts/shared

```diff
@@ -176,6 +177,13 @@ $tar $tarfile $distrobase/include/nvector/nvector_trilinos.h
 $tar $tarfile $distrobase/include/nvector/trilinos/SundialsTpetraVectorInterface.hpp
 $tar $tarfile $distrobase/include/nvector/trilinos/SundialsTpetraVectorKernels.hpp
 
+$tar $tarfile $distrobase/include/sunadaptcontroller/sunadaptcontroller_expgus.h
```

Comment:
This should be removed since it no longer exists (this is causing the Jenkins failure).